docs/blog/2024-07-17-release-1.0.0.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: release-1.0.0
+title: Release 1.0.0
+authors: [netdpb]
+tags: []
+---
+
+We are pleased to announce that
+[JSpecify 1.0](https://repo.maven.apache.org/maven2/org/jspecify/jspecify/1.0.0/)
+is available in Maven Central! That means that the four nullness annotations
+that we have defined
+([`@Nullable`](https://jspecify.dev/docs/api/org/jspecify/annotations/Nullable.html),
+[`@NonNull`](https://jspecify.dev/docs/api/org/jspecify/annotations/NonNull.html),
+[`@NullMarked`](https://jspecify.dev/docs/api/org/jspecify/annotations/NullMarked.html),
+and
+[`@NullUnmarked`](https://jspecify.dev/docs/api/org/jspecify/annotations/NullUnmarked.html))
+are official, and we will never make backwards-incompatible changes to them.
+
+Check out our [usage page](/docs/usage) for instructions on how to depend on
```

Comment:
The Docusaurus failure suggests to me that we might want just /usage instead of /docs/usage here (and presumably similarly below, including for /docs/spec, even though the "Exhaustive list" appears to mention only the /docs/usage link :))

Reply:
No, the error is pointing out that it should be `/docs/using`.